spec.html

```diff
@@ -207,7 +207,7 @@ <h1>%TypedArray%.prototype.toReversed ( )</h1>
                             1. Let _from_ be ! ToString(𝔽(_length_ - _k_ - 1)).
                             1. Let _Pk_ be ! ToString(𝔽(_k_)).
                             1. Let _fromValue_ be ! Get(_O_, _from_).
-                            1. Perform ? Set(_A_, _Pk_, _kValue_, *true*).
+                            1. Perform ! Set(_A_, _Pk_, _kValue_, *true*).
```

Comment:
Actually this might throw. Will revert.
The source array could intercept the `Get` and return an invalid value.

Reply:
From spec reading [integerindexedelementget](https://tc39.es/ecma262/#sec-integerindexedelementget) and experimenting with different implementations (e.g. below). I am convinced that this `Set` should not throw. As the source TargetArray has already been validated to be a typed-array, and the target array is constructed to be the same base type and length, so the value being inserted will be a valid value at a valid index.

```js
new class MyUnit8Array extends Uint8Array {
   [0] = 'not an int'
}
// ! Uncaught TypeError: Cannot redefine property: 0
```